python/ray/tests/test_placement_group.py

```diff
@@ -611,6 +611,53 @@ class Actor:
     )
 
 
+def test_get_assigned_resources_in_pg(ray_start_cluster):
+    cluster = ray_start_cluster
+    cluster.add_node(num_cpus=3)
+    ray.init(address=cluster.address)
+
+    @ray.remote
+    def get_assigned_resources():
+        return ray.get_runtime_context().get_assigned_resources()
+
+    resources = ray.get(get_assigned_resources.options(num_cpus=1).remote())
+    assert resources == {"CPU": 1}
+
+    pg = ray.util.placement_group(bundles=[{"CPU": 3}])
+    ray.get(pg.ready())
+
+    resources = ray.get(
+        get_assigned_resources.options(
+            num_cpus=1,
+            scheduling_strategy=PlacementGroupSchedulingStrategy(placement_group=pg),
```

Comment:
Can you add a test that specifies the bundle index?

Reply:
Done